tensorbay/cli/commit.py

```diff
@@ -32,15 +32,14 @@ def _implement_commit(obj: Dict[str, str], tbrn: str, title: str) -> None:
         error(f'To commit, "{info}" must be in draft status, like "{info}#1"')
 
     dataset_client.checkout(draft_number=info.draft_number)
-    if not title:
+    if message:
+        title, description = message[0], "\n".join(message[1:])
+    else:
```

Comment:
we need to get the description from draft and put into the editor as commit hint

Reply:
`Draft` does not have `description` yet.